packages/components/CollectionView.tsx

```diff
@@ -15,24 +13,42 @@ import { GradientText } from "./gradientText";
 type CollectionViewSize = "XL" | "XS";
 export const COLLECTION_VIEW_SM_WIDTH = 124;
 export const COLLECTION_VIEW_SM_HEIGHT = 164;
-export const COLLECTION_VIEW_XL_WIDTH = 196;
-export const COLLECTION_VIEW_XL_HEIGHT = 266;
+export const COLLECTION_VIEW_XL_WIDTH = 256;
+export const COLLECTION_VIEW_XL_HEIGHT = 315;
 
 export const CollectionView: React.FC<{
   item: Collection;
   size?: CollectionViewSize;
   linkToMint?: boolean;
-}> = ({ item, size = "XL", linkToMint }) => {
-  const [, creatorAddress] = parseUserId(item.creatorId);
-  const userInfo = useNSUserInfo(item.creatorId);
+  mintState: number;
+}> = ({ item, size = "XL", linkToMint, mintState }) => {
   const navigateToCollection = useNavigateToCollection(item.id, {
     forceSecondaryDuringMint: item.secondaryDuringMint,
     forceLinkToMint: linkToMint,
   });
   const sizedStyles = useMemo(() => StyleSheet.flatten(styles[size]), [size]);
+  let maxSupply;
+  const navigateToTwitter = () => {
+    Linking.openURL(item.twitterUrl);
+  };
+  const { info } = useCollectionInfo(item.id);
+  const percentageMinted = info
+    ? Math.round(
+        (parseInt(info.mintedAmount as string, 10) * 100) /
+          parseInt(info.maxSupply as string, 10)
+      )
+    : NaN;
+  if (!info) {
+    maxSupply = item.maxSupply;
+  } else {
+    maxSupply = info.maxSupply;
+  }
```

Comment:
you can shortcut this: `maxSupply = info ? info.maxSupply : item.maxSupply`

Reply:
done